human-controls/src/MenuController.cpp

```diff
@@ -0,0 +1,77 @@
+#include "MenuController.h"
+
+MenuController::MenuController(unsigned int encoderPinClk, unsigned int encoderPinDt,
+                               unsigned int displayAddress, unsigned int displayLen, unsigned int displayWidth,
+                               unsigned int menuSize)
+{
+    this->rotaryKnob = new RotaryKnobController(encoderPinClk, encoderPinDt);
```

Comment:
Change these to not create via new, and to use constructor initializers instead:
```
: rotaryKnob(encoderPinClk, encoderPinDt), display(displayAddress, displayLen, displayWidth)
```

Reply:
Done